docs/events.md

```diff
@@ -64,38 +29,10 @@ for managing asynchronous tasks.
 
 ## Lifespan State
 
-The event handlers also accept a `state` argument, which is a dictionary
+The lifespan can also accept a `state` argument, which is a dictionary
 that can be used to share the objects between the startup and shutdown handlers,
 and the requests.
 
```

Comment:
seems to me that this becomes outdated since there wont be startup and shutdown handlers